README.md

```diff
@@ -180,6 +180,8 @@ For CoreOS, use:
 
 `-start`: Run waagent as a background process
 
+`-collect-logs [-full]`: Runs the log collector utility that produces a zip with the relevant log files useful for support. Use flag `-full` for more exhaustive log collection. 
```

Comment:
let's add a little info as to what the tool does. i'd just mention it collects logs for debugging and where it places the zip

Reply:
"Runs the log collector utility that collects relevant agent logs for debugging and stores them in the agent folder on disk. Exact location will be shown when run. Use flag `-full` for more exhaustive log collection."
What do you think of this?

Reply:
ok